CMakeLists.txt

```diff
@@ -254,12 +284,13 @@ add_executable(Jakt::jakt ALIAS jakt_stage${FINAL_STAGE})
 
 if (NOT CMAKE_SKIP_INSTALL_RULES)
   include(cmake/install-rules.cmake)
+  jakt_install_runtime(TARGET ${JAKT_TARGET_TRIPLE})
 endif()
 
 # FIXME: Remove if we decide to use CTest
 option(JAKT_BUILD_TESTING "Whether to build tests or not, default on" ON)
 
-if (JAKT_BUILD_TESTING)
+if (JAKT_BUILD_TESTING AND NOT CMAKE_CROSSCOMPILING)
```

Comment:
We could use a cmake_dependent_option to say "default to off if cross-compiling" but meh.

Reply:
I'll use your "meh" as an excuse to run away from learning about that thing :P